docs/notebooks/local_frequency_analysis.ipynb

```diff
@@ -9,9 +9,138 @@
     },
```

Comment:
<p>Je propose plutôt ceci:</p><p> </p><p>In a future release, plotting will be handled by a proper function. For now, we'll show an example in this Notebook using preliminary utilities.</p><p> </p><p><code>xhfa.local._prepare_plots</code> generates datapoints required to plot the results of the frequency analysis. If <code>log=True</code>,will it  return log-spaced x values between <code>xmin</code> and <code>xmax</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hydrologie/xhydro/pull/87/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>